build/Dockerfile

```diff
@@ -32,16 +28,19 @@ RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
     gcloud config set metrics/environment github_docker_image && \
     gcloud --version
 
-# Install Docker
-RUN curl -fsSL get.docker.com | VERSION=19.03.8 /bin/sh
+# Install Docker (client only)
+ENV DOCKERVERSION=19.03.8
+RUN curl -fsSLo docker.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKERVERSION}.tgz && \
+    echo "7f4115dc6a3c19c917f8b9664d7b51c904def1c984e082c4600097433323cf6f  docker.tgz" | sha256sum -c && \
+    tar xzvf docker.tgz --strip 1 -C /usr/local/bin docker/docker && \
+    rm docker.tgz;
 
-# Install gometalinter
-ENV GOMETALINTER_VERSION 2.0.11
-ENV ARCH amd64
-RUN curl -fsSLo gometalinter.tar.gz --compressed "https://github.com/alecthomas/gometalinter/releases/download/v$GOMETALINTER_VERSION/gometalinter-$GOMETALINTER_VERSION-linux-$ARCH.tar.gz" && \
-    echo "97d8bd0a4d024740964c7fc2ae41276cf5f839ccf0749528ca900942f656d201 gometalinter.tar.gz" | sha256sum -c && \
-    tar xf gometalinter.tar.gz -C /usr/bin --strip-components=1 --no-same-owner && \
-    rm gometalinter.tar.gz
+# Install golangci-lint
+ENV GOLANGCILINTVERSION=1.17.1
```

Comment:
Latest version is 1.27, maybe use that one?

Reply:
It's matching the existing install https://github.com/weaveworks/wksctl/blob/6bc6a860064456c1c31f9a2df775d0228a14955d/bin/go-lint#L7

Changing it would be a separate PR.